pageserve/templates/base.html

```diff
@@ -32,44 +34,58 @@
 </head>
 
 <body>
+    <div class="topnav">
+        <a class="{% block home_tab %}{% endblock %} navtab navbtn" href='/v1/'>Home</a>
+        <a class="{% block events_tab %}{% endblock %} navtab navbtn" href='/v1/events'>Events</a>
```

Comment:
You might consider renaming `home_tab` and `events_tab` something like `home_tab_is_active` and `events_tab_is_active` or similar. On first reading, it was pretty unclear that you wanted these to toggle the `"active"` class.

Reply:
Done in 91c9612